src/app/shared/navbar/searchbar/searchbar.scss

```diff
@@ -0,0 +1,62 @@
+$input-bg: #85D9E2;
+
+@mixin color-placeholder() {
+  -webkit-font-smoothing: antialiased;
+  color: white;
+}
+
+:host {
+  position: relative;
+  flex: 2;
+
+  * {
+    box-sizing: border-box;
+  }
+
+  &.expanded .search-input-container {
+    width: 100%;
+  }
+
+  .search-input-container {
+    display: block;
+    position: relative;
+    margin-left: auto;
+    height: 100%;
+    width: 200px;
+    transition: width .2s ease;
+    &:after {
+      content: '';
+      position: absolute;
+      left: 15px; top: 50%;
+      transform: translateY(-50%);
+      height: 28px;
+      width: 28px;
+      background: url('../../../../assets/img/icons/ic_search_white_24px.svg') center center no-repeat;
+    }
+  }
+
+  input {
+    background: $input-bg;
+    border: none;
+    border-radius: 2px;
+    color: white;
+    font-size: 18px;
+    height: 95%;
+    line-height: 95%;
+    padding-left: 50px;
+    position: relative;
+    transition: width .2s ease;
+    width: 100%;
+
+    /* Set placeholder text to be white */
+    &::-webkit-input-placeholder { @include color-placeholder(); } /* Chrome/Opera/Safari */
+    &::-moz-placeholder { @include color-placeholder(); } /* Firefox 19+ */
+    &:-moz-placeholder { @include color-placeholder(); } /* Firefox 18- */
+    &:ms-input-placeholder { @include color-placeholder(); } /* IE 10+ */
```

Comment:
autoprefixer is supposed to be built into the angular-cli, but I'm not sure if there's some specific configuration needed to turn it on